app/templates/services/service_submission.html

```diff
@@ -201,7 +201,7 @@ <h2 class="temporary-message-heading" id="temporary-message-heading">
         <form action="{{ url_for('.delete_draft_service', framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id ) }}" method="POST">
           <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
           {{ govukButton({
-            "text": "Delete",
+            "text": "Delete draft service",
```

Comment:
I think this change. This is much clearer 👏

Reply:
I'll run it past Nora to make sure.

Reply:
Nora's suggested a change - I've added a couple of commits.